README.md

```diff
@@ -1,26 +1,97 @@
-<!--
-Avoid using this README file for information that is maintained or published elsewhere, e.g.:
+<div align="center">
 
-* metadata.yaml > published on Charmhub
-* documentation > published on (or linked to from) Charmhub
-* detailed contribution guide > documentation or CONTRIBUTING.md
+# CephFS server proxy operator
 
-Use links instead.
--->
+A [Juju](https://juju.is) operator for proxying exported Ceph filesystems.
 
-# cephfs-proxy-operator
+[![Charmhub Badge](https://charmhub.io/cephfs-server-proxy/badge.svg)](https://charmhub.io/cephfs-server-proxy)
+[![CI](https://github.com/canonical/cephfs-server-proxy-operator/actions/workflows/ci.yaml/badge.svg)](https://github.com/canonical/cephfs-server-proxy-operator/actions/workflows/ci.yaml/badge.svg)
+[![Release](https://github.com/canonical/cephfs-server-proxy-operator/actions/workflows/release.yaml/badge.svg)](https://github.com/canonical/cephfs-client-operator/actions/workflows/release.yaml/badge.svg)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#ubuntu-hpc:matrix.org)
 
-Charmhub package name: operator-template
-More information: https://charmhub.io/cephfs-proxy-operator
+</div>
 
-Describe your charm in one or two sentences.
+## Features
```

Comment:
You don't need this heading. The paragraph below will fit nicely with the status badges above it.

```suggestion
```